dashboards/google-home-analytics/google-home-analytics.json

```diff
@@ -0,0 +1,742 @@
+                      "crossSeriesReducer": "REDUCE_SUM",
+                      "groupByFields": []
+                    }
+                  },
+                  "unitOverride": "",
+                  "outputFullDuration": false
+                },
+                "plotType": "LINE",
+                "legendTemplate": "Cloud",
+                "minAlignmentPeriod": "3600s",
+                "targetAxis": "Y1",
+                "dimensions": [],
+                "measures": [],
+                "breakdowns": []
+              },
+              {
+                "timeSeriesQuery": {
+                  "timeSeriesFilter": {
+                    "filter": "metric.type=\"actions.googleapis.com/smarthome_action/request_count\" resource.type=\"assistant_action_project\" metric.label.\"integration_type\"=\"LOCAL\" metric.label.\"trait\"!=one_of(\"UNKNOWN\",\"UnknownTrait\")",
+                    "aggregation": {
+                      "alignmentPeriod": "3600s",
+                      "perSeriesAligner": "ALIGN_DELTA",
+                      "crossSeriesReducer": "REDUCE_SUM",
+                      "groupByFields": []
+                    }
+                  },
+                  "unitOverride": "",
+                  "outputFullDuration": false
+                },
+                "plotType": "LINE",
+                "legendTemplate": "Local",
+                "minAlignmentPeriod": "3600s",
+                "targetAxis": "Y1",
+                "dimensions": [],
+                "measures": [],
+                "breakdowns": []
+              },
+              {
+                "timeSeriesQuery": {
+                  "timeSeriesFilter": {
+                    "filter": "metric.type=\"actions.googleapis.com/smarthome_action/request_count\" resource.type=\"assistant_action_project\" metric.label.\"integration_type\"=\"MATTER\" metric.label.\"trait\"!=one_of(\"UNKNOWN\",\"UnknownTrait\")",
+                    "aggregation": {
+                      "alignmentPeriod": "3600s",
+                      "perSeriesAligner": "ALIGN_DELTA",
+                      "crossSeriesReducer": "REDUCE_SUM",
+                      "groupByFields": []
+                    }
+                  },
+                  "unitOverride": "",
+                  "outputFullDuration": false
+                },
+                "plotType": "LINE",
+                "legendTemplate": "Matter",
+                "minAlignmentPeriod": "3600s",
+                "targetAxis": "Y1",
+                "dimensions": [],
+                "measures": [],
+                "breakdowns": []
+              }
+            ],
+            "thresholds": [],
+            "yAxis": {
+              "label": "",
+              "scale": "LINEAR"
+            },
+            "chartOptions": {
+              "mode": "COLOR",
+              "showLegend": false,
+              "displayHorizontal": false
+            }
+          },
+          "title": " Fulfillment Request Count",
+          "id": ""
```

Comment:
Can we remove all of the vestigial empty `id` fields? I get the sense they should be specific to an instance of a dashboard, so don't really belong in a template.

Here and on other dashboards.

Reply:
Done!